Sami/RemoteRepository/GitHubRemoteRepository.php

```diff
@@ -15,7 +15,7 @@ class GitHubRemoteRepository extends AbstractRemoteRepository
 {
     public function getFileUrl($projectVersion, $relativePath, $line)
     {
-        $url = 'https://github.com/'.$this->name.'/blob/'.$projectVersion.$relativePath;
+        $url = 'https://github.com/'.$this->name.'/blob/'.str_replace('\\', '/',$projectVersion.$relativePath);
```

Comment:
1. Please use `DIRECTORY_SEPARATOR` constant (PHP build-in) instead of `'\\'`.
2. There should be 1 space after comma in argument list passed to function.